utils/led.c

```diff
@@ -6,24 +6,19 @@
  */
```

Comment:
Can you please add the GPL header (copy over from other files)?

Reply:
yes, done.